### DISCUSSION TITLE
Disqus Comments

### DISCUSSION THREAD

**Discussion by 0xtiago** [gethinode/hinode#1071] [Help] [answered]
Hugo has a builtin support to Disqus, but I couldnt figure out how to make it work in Hinode. I have enable the service in hugo.toml, configured server.toml e even created a partial called disqus.html in layouts/partials with the content of this reference https://gohugo.io/templates/embedded/#disqus ...with no success.
Reading the text above you can see that I followed everything to this [previous discussion](https://github.com/gethinode/hinode/discussions/454). 
Anybody had the same problem like me?

Tks!

**Comment by markdumay** [comment, marked as answer]
Most likely you're running into issues with the Content Security Policy. Hinode uses [strict policies by default](https://gethinode.com/docs/advanced-settings/server-headers/). You can view the issues in your browser. For example, in Chrome, right-click on the page and select `Inspect` to open the debugging tools. It will show you all warnings and errors.

I've set up a test repository with a relaxed policy here: https://github.com/markdumay/hinode-disqus

You can review the changes I made to the base template in this PR: https://github.com/markdumay/hinode-disqus/pull/2

**Reply by markdumay** [reply]
One additional remark, you will also need to update the Content Security Policy of your deployment server. You can use the [local Hugo server configuration](https://raw.githubusercontent.com/markdumay/hinode-disqus/main/config/_default/server.toml) as a reference.

**Comment by 0xtiago** [comment]
Hi @markdumay! The only (the main) thing that I've havent seen anywhere was to create the _default/single.html, maybe I miss this part. Everything is working as you can see : https://tiagotavares.io/blog/gestao-de-vulnerabilidades-com-greenbone-openvas_dez2021/

Again, thank you very much!

**Reply by markdumay** [reply]
Glad it works now! The Hugo docs are not very explicit how / where to add the template: https://gohugo.io/content-management/comments/#render-hugos-built-in-disqus-partial-template
